
# HG changeset patch
# User Darren Salt <linux@youmustbejoking.demon.co.uk>
# Date 1268758696 0
# Node ID a68dd15ff7ae4be65c2ace25ef66966d7c272514
# Parent 882c464d4a650d09da37ade366face74452c1897
Restore definition of HAVE_LIRC for use in makefiles.

--- a/m4/_xine.m4	Mon Mar 15 12:03:01 2010 -0700
+++ b/m4/_xine.m4	Tue Mar 16 16:58:16 2010 +0000
@@ -62,6 +62,7 @@
      if test "$found_lirc" = yes; then
 	AC_DEFINE([HAVE_LIRC],,[Define this if you have LIRC (liblirc_client) installed])
      fi
+     AM_CONDITIONAL([HAVE_LIRC], [test "$found_lirc" = yes])
      AC_SUBST(LIRC_LIBS)
      AC_SUBST(LIRC_INCLUDE)
 ])

